_posts/2019-01-24-how-to-keep-safe-as-a-client.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title:  "how to keep safe as a client"
+date:   2019-01-27 22:00:00 +0800
+categories: blog
+author: Joe Zhang
+---
+
+Recently I am working on a project called transcript, It will use many third party services, generally third party services will expose restful APIs and my project will act as a client.
+
+![](https://github.com/BarkZhang/integration-blog/blob/master/assets/2019-01-24-how-to-keep-safe-as-a-client/transcript_sample.png)
+
+In such background, If some services broken down or having high latency, It will affect our system. what's more, It will block user request and may cause the whole system broken down.
```

Comment:
We should use initial capital right after period(.) and lower case after comma(,)
In such background, if some services broken down or having high latency, it will affect our system. What's more, it will block user request and may cause the whole system broken down.